linode/helper/conversion.go

```diff
@@ -62,6 +62,14 @@ func StringToInt(s string, diags *diag.Diagnostics) int {
 	return num
 }
 
+func FrameworkSafeInt64PointerToIntPointer(numberPtr *int64, diags *diag.Diagnostics) *int {
```

Comment:
Just to shorten this function name a bit, would it make sense to call it something like `FrameworkSafeInt64PtrToIntPtr`?